packages/get-platform/src/getPlatform.ts

```diff
@@ -264,16 +365,16 @@ export async function getSSLVersion(args: GetOpenSSLVersionParams): Promise<GetO
 }
 
 export async function getPlatform(): Promise<Platform> {
-  const { platform, distro, arch, libssl } = await getos()
+  const { platform, arch, libssl, targetDistro, familyDistro, originalDistro } = await getos()
 
   // sometimes we fail to detect the libssl version to use, so we default to 1.1.x
   const defaultLibssl = '1.1.x' as const
   if (platform === 'linux' && libssl === undefined) {
     /**
-     * Ask the user to install openssl manually, and provide some additional instructions based on the detected Linux distro.
+     * Ask the user to install openssl manually, and provide some additional instructions based on the detected Linux distro family.
      */
-    const additionalMessage = match({ distro })
-      .with({ distro: 'debian' }, () => {
+    const additionalMessage = match({ familyDistro })
+      .with({ familyDistro: 'debian' }, () => {
         return "Please manually install OpenSSL via `apt-get update -y && apt-get install -y openssl` and try installing Prisma again. If you're running Prisma on Docker, you may also try to replace your base image with `node:lts-slim`, which already ships with OpenSSL installed."
       })
```

Comment:
`familyDistro` is used in this warning so that we can tell the user which package manager to use when installing e.g. `openssl`